src/containers/ArticlePage/components/Comment.tsx

```diff
@@ -78,34 +78,30 @@ const TopButtonRow = styled.div`
 export type CommentType = { generatedId?: string; content: string; isOpen: boolean } | IComment;
 
 interface Props {
-  comment: CommentType;
-  allOpen: boolean | undefined;
-  setAllOpen: (v: boolean | undefined) => void;
   comments: CommentType[];
   setComments: (c: CommentType[]) => void;
   onDelete: (index: number) => void;
   index: number;
+  setCommentsOpen: (commentsOpen: boolean[]) => void;
+  commentsOpen: boolean[];
 }
 
 const Comment = ({
-  comment,
-  allOpen,
-  setAllOpen,
   comments,
   setComments,
   onDelete,
   index,
```

Comment:
Berre et spørsmål. Du sender inn alle comments + index inn i kvar Comment for å vise fram den med korrekt indeks. Virker litt overkill, men det gjør det kanskje lett å håndtere alle samtidig.